docs/stakers/cluster-management/update-operators.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Update Operators
+sidebar_position: 5
+---
+
+# Update Operators
+
+:::warning Collusion risks
+Please do not change more than 2 operators in a cluster. Each set of **generated shares will always be valid** when their signing threshold is met (e.g. 3/4). To reduce the risks, it is advised to not change more than 2 of the validator's managing operators when changing its cluster.
+:::
+
+Before we jump into step-by-step instructions, we need to verify how your validator was generated. Depending on that fact, your steps will differ:
```

Comment:
Using "we" is more of a "YouTube" guide kind of thing. In official docs it's always better to remain impersonal with "Before jumping in....it's necessary to verify" or direct assertive phrases in the imperative mode like "do this, do that", "you need to do this/that".
For example, the last sentence: "Depending on that fact, your steps will differ" is great.

As much as subjective this may sound (which in small part it is, but it's also a generally established rule for this kind of literature), it's also for the sake of consistency, since the rest of the docs are written this way, and also the majority of this page as well is.